roles.md

```diff
@@ -2,8 +2,125 @@
 layout: default
 ---
 
-### Roles
+# Roles
 
-## The Good Docs Project Steering Committee
+Here we define roles within The Good Docs Project, along with the eligibility, expectations, and responsibilities associated with these roles.
 
-*   Information about the Steering Committee can be found [here](https://github.com/thegooddocsproject/governance/blob/master/ProjectSteeringCommittee.md).
+**Version:** 2.0
+
+**Last updated:** August 2020
+
+## Community
+
+Everyone contributing to, or using artifacts from, The Good Docs Project is considered part of our community. We encourage all community members to actively participate in our activities. This includes:
+
+*   Participate in our forums and vote on motions.
+*   Raise and comment on bugs.
+*   Suggest fixes by raising pull requests.
+
+**Access:**
+
+*   Community members are granted _read_ access to our github repositories.
+
+**Expectations of community members:**
+
+*   Community members are expected to be respectful to each other, in line with our [Code of Conduct](https://github.com/thegooddocsproject/governance/blob/master/CodeOfConduct.md).
+
+## Committers
+
+A committer is a trusted community member who has additionally been given _write_ access to some or all of our github repositories.
+
+**Access:**
+
+*   Committers are granted _triage_, _maintain_ and _write_ access to the github repositories which they have contributed to.
+
+**Additional expectations of committers:**
+
+*   Understand and apply our quality guidelines.
+*   Understand and abide by our license requirements.
+*   Periodically contribute to our project over a one year period.
+
+**Eligibility:**
+
+Before being accepted for this role, a person should display the following characteristics:
+
+*   Provide sustained, valuable contributions to the project, typically for three months or more. An example demonstration of this could be:
+    *   Submit pull requests which have been accepted and merged by other committers. Typically one large, three medium, or five small pull requests over three months.
```

Comment:
Do we need to define what large, medium, and small PRs are? I guess small is typos, and large is structural changes, but not sure what medium might be.

Reply:
Definitely something to capture as part of iteration 2 (or even I3) of this document.

Reply:
Also, track people's contributions from the discussion side -- opening/commenting on  `Issues` and on PRs.  As not every contributor is going to be able to open a PR.  This means we aren't inclusive of less-experience individuals looking to become members.... Not sure if this means we change this from `Committers` to `Contributors`...